stride/assetlist.json

```diff
@@ -686,6 +684,45 @@
           }
         }
       ]
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "stuband",
+          "exponent": 0
+        },
+        {
+          "denom": "stBAND",
+          "exponent": 6
+        }
+      ],
+      "base": "stuband",
+      "name": "Stride Staked BAND",
+      "display": "stBAND",
+      "symbol": "stBAND",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "laozi-mainnet",
```

Comment:
Please correct the chain_name to "bandchain"

Reply:
good catch! this should be fixed now